ark/phenotyping/som_utils.py

```diff
@@ -937,13 +943,50 @@ def create_pixel_matrix(fovs, channels, base_dir, tiff_dir, seg_dir,
     if not os.path.exists(os.path.join(base_dir, subset_dir)):
         os.mkdir(os.path.join(base_dir, subset_dir))
 
+    # create variable for storing 99.9% values
+    quant_dat = pd.DataFrame()
+
+    # define a path for storing the intermediate quant_dat values, needed in case of restart
+    # TODO: change this to saving in the pixel_output_dir after merging in master
+    quant_dat_path = os.path.join(base_dir, pixel_output_dir, 'quant_dat.feather')
```

Comment:
What is the scenario where creating the 99.9% normalized values would fail? We haven't generated the feather files yet. What type of error is this guarding against?

Reply:
Let's say the user runs a cohort of 100 FOVs, and `create_pixel_matrix` crashes 97 FOVs in. Previously, we would have lost any reference to the 97 FOVs in `quant_dat`. On restart, `quant_dat` would only contain the data for the last 3 FOVs, which would affect the calculation of `mean_quant` and 99.9% normalization prior to training the SOM.

Intermediate saving of `quant_dat` helps guard against this. It's mostly to ensure correctness, but I'll remove it if you don't anticipate this being an issue.